web/src/components/table/simpleTable.js

```diff
@@ -356,7 +271,7 @@ export default function SimpleTable({
       )}
       <Grid item xs={12} lg={12}>
         <DataTable
-          header={header}
+          header={isglobalfilter ? header : null}
```

Comment:
isglobalfilter should be camelcase =>  isGlobalFilter

Reply:
Updated